sites/platform/src/create-apps/app-reference.md

```diff
@@ -193,50 +193,152 @@ For more information, see how to [manage resources](/manage-resources.md).
+and connects your app to it through the service container's default endpoint.
+
+{{< note title="Tip">}}
+
+An even quicker way to define many relationships is to use the following single-line configuration:
+
+```yaml {configFile="app"}
+{{< snippet>}}
+relationships: {{{< variable "SERVICE_NAME_A" >}}, {{< variable "SERVICE_NAME_B" >}}}
+{{< /snippet>}}
+```
+{{< /note >}}
+
+You can add as many relationships as you want to your app configuration:
+
+```yaml {configFile="app"}
+{{< snippet>}}
+relationships:
+    mariadb:  
+    redis: 
+    elasticsearch: 
+{{< /snippet>}}
+```
+
 <--->
+
++++
+title=Using explicit endpoints
++++
+
+Use the following configuration:
+
 ```yaml {configFile="app"}
-applications:
-    myapp:
-        source:
-            root: "/"
-        type: 'nodejs:{{% latest "nodejs" %}}'
-        relationships:
-            database: 'mysqldb:db1'
-            database2: 'mysqldb:db2'
-            cache: 'rediscache:redis'
-            search: 'searchserver:elasticsearch'
+{{< snippet >}}
+relationships:
+    {{% variable "RELATIONSHIP_NAME" %}}:
+        service: {{% variable "SERVICE_NAME" %}} 
+        endpoint: {{% variable "ENDPOINT_NAME" %}}
+{{< /snippet >}}
 ```
-{{% /version/specific %}}
+ 
+- `RELATIONSHIP_NAME` is the name you want to give to the relationship.
+- `SERVICE_NAME` is the name of the service as defined in its [configuration](/add-services/_index.md).
+- `ENDPOINT_NAME` is the endpoint to connect to the service.
+
+For example, to define a relationship named `database` that connects your app to a service called `mariadb` through the `db1` endpoint,
+use the following configuration:
+
+```yaml {configFile="app"}
+{{< snippet >}}
+relationships:
+    database: # The name of the relationship. 
+        service: mariadb
+        endpoint: db1
+{{< /snippet >}}
+```
+
+For more information on how to handle multiple databases, multiple cores,
+and/or different permissions with services that support such features,
+see each service's dedicated page:
+
+ - [MariaDB/MySQL](/add-services/mysql/_index.md#multiple-databases) (multiple databases and permissions)
+ - [PostgreSQL](/add-services/postgresql/_index.md#multiple-databases) (multiple databases and permissions)
+ - [Redis](/add-services/redis/_index.md#multiple-databases) (multiple databases)
+ - [Solr](add-services/solr/_index.md#solr-6-and-later) (multiple cores)
+ - [Vault KMS](add-services/vault/_index.md#multiple-endpoints-example) (multiple permissions)
+
+ You can add as many relationships as you want to your app configuration:
+
+```yaml {configFile="app"}
+{{< snippet>}}
+relationships:
+    database1: 
+        service: mariadb
+        endpoint: admin
+    database2:
+        service: mariadb
+        endpoint: legacy
+    cache:
+        service: redis
+        endpoint: redis
+    search:
+        service: elasticsearch
+        endpoint: elasticsearch
```

Comment:
Maybe for `cache` and `search` use the simplified notation, to show that it can be mixed with the explicit one